packages/data/src/redux-store/index.js

```diff
@@ -601,6 +601,9 @@ function mapSelectorWithResolver(
 	}
 
 	const selectorResolver = ( ...args ) => {
+		if ( resolver.normalizeArgs && args?.length ) {
+			args = resolver.normalizeArgs( args );
```

Comment:
might be worth documenting this new API. Also should we also call normalize in the selector function?

Reply:
Definitely needs documenting 👍

Reply:
I've now "documented" this in this PR but I don't think I've done a great job thus far. Needs improving. Input welcomed.